fusion/imagel_single[] image_points
float64 stamp
